scripts/java-api-check.sh

```diff
@@ -157,4 +159,12 @@ compare_aars() {
 api_compat=$(compare_aars "${PREVIOUS_RELEASE}" "${CURRENT_RELEASE}")
 rm -rf "${TMPDIR}"
 echo "Compare result: $api_compat"
+
+#if [[ -z ${TAGGED_RELEASE_VERSION} ]]; then
```

Comment:
Shall we enable this to fail the CI job when finding api breaking change?

Reply:
imho yes
however that CI check should not prevent PR to be merged

also would be nice if API break logs could be easily visible in CI logs

tagging @mapbox/maps-android as well to collect feedback

Reply:
Why shouldn't it prevent to merge PR though?

Reply:
@yunikkk We will never be able to merge new PR with api-breaking even we decide to bump the major version number.

Reply:
Hmm. I'd suggest to update the script logic so that it would NOT check api breaking changes on major update, only minor/patch. WDYT?

Reply:
In that way, we will not get notified if a PR contains api breaking changes.

Reply:
Not sure I understand, could you add more details why?

Reply:
As discussed below, the api check job is only a notification, will not block any other jobs or merge a PR, so it doesn't matter we keep check for major update.